app/controllers/users/registrations_controller.rb

```diff
@@ -5,6 +5,7 @@ class Users::RegistrationsController < Devise::RegistrationsController
   def create
     @user = User.new(configure_sign_up_params)  
     if @user.save
+      sign_up(resource_name, resource)
       render json: @user
```

Comment:
@josephelias-rubyians  Hi Joseph, lets use the serialiser in this object.

Reply:
I didn't understand this. I think  sign_up(resource_name, resource) is a helper method for sign up and generating token.

Reply:
@josephelias-rubyians  Sorry, I added the comment for line 9. render json: @user